src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -32,6 +33,36 @@
  */
 public class OptionData implements SerializableData
 {
+    /**
+     * The highest positive amount Discord allows the {@link OptionType#NUMBER NUMBER} type to be.
+     */
+    public static final double MAX_POSITIVE_NUMBER = (1L << 53);
+    
+    /**
+     * The smallest negative amount Discord allows the {@link OptionType#NUMBER NUMBER} type to be.
+     */
+    public static final double MIN_NEGATIVE_NUMBER = -(1L << 53);
```

Comment:
Same as above... or below... this is confusing.

Numbers this size or more negative than this will fail:
![image](https://user-images.githubusercontent.com/5003579/129821708-c664eaa9-1fcf-4af5-a5b9-a8d2f3109b9a.png)

Numbers less negative than this will succeed:
![image](https://user-images.githubusercontent.com/5003579/129821732-e3cb59bb-d65f-47d5-927e-85376e8c019c.png)

Reply:
Also can I suggest changing the comment to read "most negative amount", or "largest negative value" or similar?

Reply:
> Also can I suggest changing the comment to read "most negative amount", or "largest negative value" or similar?
Good point. It isn't a "small" negative number.